domain/ide/workspace/workspace_test.go

```diff
@@ -73,7 +73,7 @@ func Test_TrustFoldersAndScan_shouldAddFoldersToTrustedFoldersAndTriggerScan(t *
 	}, time.Second, time.Millisecond, "scanner should be called after trust is granted")
 }
 
-func Test_AddAndRemoveFoldersAndTriggerScan(t *testing.T) {
+func Test_AddAndRemoveFoldersAndReturnFolderList(t *testing.T) {
```

Comment:
Where do you test the extracted trigger scan path now?

Reply:
in server_test Test_workspaceDidChangeWorkspaceFolders_shouldProcessChanges it checks if folder.Scanned() 
I think it should be enough, wdyt ?